airbyte-local.sh

```diff
@@ -344,4 +346,44 @@ function err() {
     exit 1
 }
 
+main() {
+    setDefaults
+    parseFlags "$@"
+    validateInput
+    tempdir=$(mktemp -d)
+    trap cleanup EXIT
+    trap cleanup SIGINT
+    echo "Created folder $tempdir for temporary airbyte files"
+
+    if ((no_src_pull)); then
+        log "Skipping pull of source image $src_docker_image"
+    else
+        log "Pulling source image $src_docker_image"
+        docker pull $src_docker_image
+    fi
+    writeSrcConfig
+    writeSrcCatalog
+
+    if ((no_dst_pull)); then
+        log "Skipping pull of destination image $dst_docker_image"
+    else
+        log "Pulling destination image $dst_docker_image"
+        docker pull $dst_docker_image
+    fi
+    parseStreamPrefix
+    writeDstConfig
+    writeDstCatalog
+    
+    checkSrc
+    loadState
+    if ((run_src_only)); then
+        log "Only running source"
+        readSrc
+    else
+        log "Running source and passing output to destination"
+        sync
+    fi
+    log "Done"
```

Comment:
Thought this would be nice cause its not super obvious when the destination is done.